endpoints/rfps.yaml

```diff
@@ -0,0 +1,92 @@
+openapi: "3.0.0"
+
+info:
+  version: 1.0.0
+  title: RFP
+  description: Allows the buyer to submit a Request for Proposal (RFP) to the seller with overall media budget, goal and makegood terms and optionally include specific detailed inventory requests
+  termsOfService: http://placeholderdomain.io/terms/
+  contact:
+    name: TIP Initiative
+    email: tipinitiative@frontrowadvisory.com
+    url: http://placeholderdomain.io
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+servers:
+  - url: http://placeholderdomain.io
+
+components:
+ securitySchemes:
+    bearerAuth: 
+      type: http
+      scheme: bearer
+      bearerFormat: JWT 
+
+security:
+  - bearerAuth: []
+
+paths:
+  /rfps:
+    post:
+      summary: Create an RFP in the seller's system
+      operationId: createRFP
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: 'schemas/rfpSchemas.yaml#/components/schemas/rfpRequestObject'
+      responses:
+        '200':
+          description: request processed successfully
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:                
+                $ref: 'schemas/commonSchemas.yaml#/components/schemas/Error'                
+
+  /rfps/{rfpId}:
+    parameters:
+    - name: rfpId
+      in: path
+      required: true
+      description: The RFP Id
+      schema:
+        type: string
+
+    patch:
+      summary: Change the details of a RFP in the seller's system
+      operationId: updateRFP
+      requestBody:
+        required: true # TODO: What should the request body look like for a PATCH?
+        content:
+          application/json:
+            schema:
+              $ref: 'schemas/rfpSchemas.yaml#/components/schemas/rfpRequestObject'
+      responses:
+        '200':
+          description: request processed successfully
+        '404':
+          description: RFP not found
+        default:
+          description: unexpected error
+          content:
+            application/json:    
+              schema:
+                $ref: 'schemas/commonSchemas.yaml#/components/schemas/Error'        

```

Comment:
Should the PATCH endpoint object have the same required fields as the main POST endpoint? I can definitely see `transactionHeader` and `referenceId` being required, but having all the other required fields needed in a PATCH context seems unneeded. Any thoughts?

Reply:
We're not sure that a PATCH will be needed. Typically today an RFP is just resent if it needs to be superceded. That might suggest a PUT and DELETE are needed. Checking...